ui/desktop/app/components/card/index.hbs

```diff
@@ -0,0 +1,17 @@
+<section class='card'>
+  <header class='card-header'>
+    <div>
+      {{#if @icon}}
+        <Rose::Icon @name={{@icon}} @size='medium' />
+      {{/if}}
+      {{@heading}}
+    </div>
+    {{yield (hash header=(component 'rose/anonymous'))}}
```

Comment:
When we're just defining an empty region, let's use named blocks, e.g.
```
{{yield to="header"}}
```

Where developers will use it thusly:
```
<Card>
  <:header>This is my header!</:header>
</Card>
```

Reply:
Also I think by default, the header region should be right-aligned to reflect the design.